app/views/posts/index.html.erb

```diff
@@ -24,58 +24,61 @@
       <% end %>
     </div>
   </div>
-  <div class="col-md-12">
-    <%= grid(@post_grid) do |g|
-      g.column name: "Titel", attribute: "title", filter: false do |post|
-        link_to(post.title, %(##{dom_id(post)}), class: "", data: {toggle: "collapse"})
-      end
-      g.column name: "Gräns", attribute: "limit", filter: false do |post|
-        (post.limit) && (post.limit > 0) ? post.limit : "Ingen"
-      end
+  <% if @post_grid.present? && @posts.present? && @posts.count > 0 %>
+    <div class="col-md-12">
+      <%= grid(@post_grid) do |g|
+        g.column name: "Titel", attribute: "title", filter: false do |post|
+          link_to(post.title, %(##{dom_id(post)}), class: "", data: {toggle: "collapse"})
+        end
+        g.column name: "Gräns", attribute: "limit", filter: false do |post|
```

Comment:
These attributes already have translations [here](https://github.com/fsek/web/blob/master/config/locales/models/post.sv.yml). Don't repeat yourself!

Label tags translate attributes automatically. For other cases, you use ```human_attribute_name```:

http://apidock.com/rails/ActiveModel/Translation/human_attribute_name